recipes-debian/drbd-utils/drbd-utils_debian.bb

```diff
@@ -0,0 +1,165 @@
+#base recipe: meta-openembedded/meta-networking/recipes-support/drbd/drbd-utils_9.5.0.bb
+#base branch: thud
+#base commit: 446bd615fd7cb9bc7a159fe5c2019ed08d1a7a93
+
+SUMMARY = "Distributed block device driver for Linux"
+DESCRIPTION = "DRBD mirrors a block device over the network to another machine.\
+DRBD mirrors a block device over the network to another machine.\
+Think of it as networked raid 1. It is a building block for\
+setting up high availability (HA) clusters."
+HOMEPAGE = "http://www.drbd.org/"
+SECTION = "admin"
+LICENSE = "GPLv2+"
+LIC_FILES_CHKSUM = "file://COPYING;md5=5574c6965ae5f583e55880e397fbb018"
+
+# v9.5.0
+SRCREV_drbd-utils = "ee126652638328b55dc6bff47d07d6161ab768db"
+SRCREV_drbd-headers = "b47cc11bcabe1a65c40ad23f71dcaf2da6419630"
```

Comment:
Do we need SRCREV_* variables:?

Reply:
As you say, it is unnecessary. Removed it.